### ISSUE TITLE
new release?

### ISSUE THREAD

**Issue by agitator** [plone/plone.formwidget.recaptcha#9]
@tisto @davisagli

**Comment by tisto** [comment]
Will look into it tomorrow...

**Comment by instification** [comment]
+1 for this. We are using this with collective.easyform but it's broken on 2.0a3 (latest release as of this post) and working for master.

Culprit seems to be https://github.com/plone/plone.formwidget.recaptcha/pull/7 which is merged but unreleased.

**Comment by tisto** [comment]
We have to fix Travis first. Will look into it...

**Comment by instification** [comment]
@tisto no problem, thanks for that.

FWIW I had a look at building locally from the buildout.cfg contained in the repo and found that I had to install the following with pip:

```
zc.buildout==2.9.4
setuptools==33.1.1
```

I also changed `buildout.cfg` to `extends=test-5.0.cfg` and added `package-path = ${buildout:directory}/src/plone/formwidget/recaptcha` to that `test-5.0.cfg`. Then I was able to build it locally.

Not sure if this is of any use to you :)
